backend/entityservice/api_def/swagger.yaml

```diff
@@ -509,25 +509,26 @@ paths:
 
         ### result_type = "similarity_scores"
 
-        The list of the indices of potential matches and their similarity score
+        The list of the candidate of potential matches and their similarity score
```

Comment:
How about: "The list of the pairwise similarity scores where the score is greater than the similarity threshold."
(when we create a run we talk about 'similarity threshold', not about 'mapping threshold'.)
I don't think we should talk about potential matches here, as we return everything.

Reply:
:+1: